### ISSUE TITLE
geckodriver not available in node browser images. 

### ISSUE THREAD

**Issue by levlaz** [circleci/circleci-images#125]
On `circleci/node:8-browsers` we see: 

```
      Error: The geckodriver executable could not be found on the current PATH. Please download the latest version from https://github.com/mozilla/geckodriver/releases/ and ensure it can be found on your PATH.
          at findGeckoDriver (/home/circleci/code/node_modules/selenium-webdriver/firefox/index.js:354:11)
          at new ServiceBuilder (/home/circleci/code/node_modules/selenium-webdriver/firefox/index.js:446:22)
          at Function.createSession (/home/circleci/code/node_modules/selenium-webdriver/firefox/index.js:516:21)
          at createDriver (/home/circleci/code/node_modules/selenium-webdriver/index.js:170:33)
          at Builder.build (/home/circleci/code/node_modules/selenium-webdriver/index.js:645:16)
          at fn (/home/circleci/code/lib/run-test.js:203:52)
          at <anonymous>
          at process._tickCallback (internal/process/next_tick.js:188:7)
```

I think it makes total sense to include this in any images that have the `browser` tag. 

Short term workaround is to install it as a part of the build with: 

```
sudo wget https://github.com/mozilla/geckodriver/releases/download/v0.19.1/geckodriver-v0.19.1-linux64.tar.gz
sudo tar xf geckodriver-v0.19.1-linux64.tar.gz -C /usr/local/bin
```

**Comment by iynere** [comment]
pr here ! https://github.com/circleci/circleci-images/pull/126

**Comment by iynere** [comment]
@levlaz im gonna undo this change & take geckodriver back out, it's not compatible with the version of firefox we're installing in our images & doesn't sound like we're ready to upgrade to firefox 53+ yet (folks are worried it might break other things in the images)

**Comment by jonathanong** [comment]
@iynere interesting. any insight on why you're not ready for FF 53+?

**Comment by iynere** [comment]
@jonathanong nothing in particular, i don't think—we just don't wanna quickly bump it w/o looking into it, so we don't end up breaking a bunch of things. i'll try to run some tests w/ FF53 next week.

**Comment by stephenmathieson** [comment]
Any progress or updates here?

**Comment by stephenmathieson** [comment]
Great! Glad to hear it =]

Please keep us updated.

**Comment by jkodu** [comment]
Is there a fix for this? Any heads-up appreciated.

**Comment by bwagnerr** [comment]
Bumping this, we were using the gecko driver (as it was available on the ruby 2.5 node browsers docker image), but it seems that last week all images were re-generated and our tests started breaking because there is no gecko driver anymore.

**Comment by iynere** [comment]
thank you for the feedback ! we will get to this soon i promise, we appreciate your patience

also, this is an open-source repo, so please feel free to dive into the code here and submit a PR—& happy to answer any questions / review any potential implementations, etc.
